src/openst.js

```diff
@@ -70,6 +70,7 @@ class OpenST {
     };
     Object.assign(this.chainConfig.openst, setupData);
     logger.info('Completed Setup of OpenST');
+    return setupData;
```

Comment:
I don't think you need this change 🤔 The contents of `setupData` are available in `chainConfig.openst` via L71, right?

Reply:
Yes it's written to config as part of setup.
Removed the above return.